app-text/tesseract/tesseract-3.04.00-r4.ebuild

```diff
@@ -92,28 +103,41 @@ src_unpack() {
 		 -execdir sh -c 'cp -- "$0" "${S}/tessdata/${0%-*}"' '{}' ';' || die
 }
 
+src_prepare() {
+	epatch "${PATCHES[@]}"
+	eautoreconf
+
+	java-pkg-opt-2_src_prepare
+}
+
 src_configure() {
-	local myeconfargs=(
+	econf \
```

Comment:
I didn't know that was a thing and thought we only needed arrays for passing into eclass functions. Is this just your personal stylistic nitpick? :stuck_out_tongue: